proxy/coco/src/request.rs

```diff
@@ -316,12 +316,12 @@ impl<T> Request<Cloning, T> {
     /// This signifies that the clone was successful and that the whole request was successful,
     /// congratulations.
     #[allow(clippy::use_self, clippy::missing_const_for_fn)]
-    pub fn cloned(self, repo: RadUrn, timestamp: T) -> Request<Cloned, T> {
+    pub fn cloned(self, timestamp: T) -> Request<Cloned, T> {
```

Comment:
It might interesting to record the url we cloned from.

Reply:
_smh_ thought we'd decided otherwise? :sweat_smile:

Reply:
Can't remember that we ever had the url as an input. It might be of interest to see what peer satisfied our request.

Reply:
Ah the url, sorry was thinking urn. `RadUrl` makes sense :+1: